CHANGELOG.md

```diff
@@ -25,13 +26,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - fix: fix scheduler metrics remote write and relabel regex [#2058][#2058]
 
 [#2036]: https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/2036
-[#2056]: https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/2056
```

Comment:
Why remove this?

Reply:
`2056`, `2065` :roll_eyes:

Reply:
@swiatekm-sumo sorry, I had too many changelog changes recently and automatically fixed this (thinking this is my another PR)

Reply:
Huh, accident while rebasing, will fix.